app/src/main/java/com/teamagam/gimelgimel/app/map/viewModel/MapViewModel.java

```diff
@@ -155,6 +163,8 @@ public void display(VectorLayerPresentation vectorLayerPresentation) {
         @Override
         public void entityClicked(String entityId) {
             mSelectEntityInteractorFactory.create(entityId).execute();
```

Comment:
We could've made an entirely new interactor here that does those two interactor calls.
I leave it up to you to decide if you want to do it now or not.